plugins/module_utils/better_arg_parser.py

```diff
@@ -329,6 +358,70 @@ def _bool_type(self, contents, resolve_dependencies):
             raise ValueError('Invalid argument "{0}" for type "bool".'.format(contents))
         return contents
 
+    def _member_name_type(self, contents, resolve_dependencies):
+        """Resolver for data_set type arguments.
+
+        Parameters
+        ----------
+        contents : bool
+            The contents of the argument.
+        resolved_dependencies : dict
+            Contains all of the dependencies and their contents,
+            which have already been handled,
+            for use during current arguments handling operations.
+
+        Returns
+        -------
+        str
+            The arguments contents after any necessary operations.
+
+        Raises
+        ------
+        ValueError
+            When contents is invalid argument type.
+        """
+        if not fullmatch(
+            r"^[A-Z$#@]{1}[A-Z0-9$#@]{0,7}$",
+            str(contents),
+            IGNORECASE,
+        ):
+            raise ValueError(
+                'Invalid argument "{0}" for type "member_name".'.format(contents)
+            )
+        return str(contents)
+
+    def _identifier_name_type(self, contents, resolve_dependencies):
+        """Resolver for data_set type arguments.
```

Comment:
Since this is used only in zos_started task could you comment a bit of where this is used ?